integration/runtime_handler_test.go

```diff
@@ -19,38 +19,40 @@
 package integration
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 )
 
-// TODO(chrisfegly): add/update test(s) to allow testing of multiple runtimes at the same time
-func TestRuntimeHandler(t *testing.T) {
-	t.Logf("Create a sandbox")
-	sbConfig := PodSandboxConfig("sandbox", "test-runtime-handler")
-
-	if *runtimeHandler == "" {
-		t.Logf("The --runtime-handler flag value is empty which results internally to setting the default runtime")
-	} else {
-		t.Logf("The --runtime-handler flag value is %s", *runtimeHandler)
+func TestRuntimes(t *testing.T) {
```

Comment:
Was thinking about a matrix of runtime configs.

See `[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc[` and `[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]` over here https://github.com/containerd/containerd/blame/master/docs/cri/config.md#L140-L209